src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/AndroidLoginPage.java

```diff
@@ -79,4 +87,52 @@ public CarinaDescriptionPageBase login() {
         return clickLoginBtn();
     }
 
+
+    public boolean isNameInputFieldArePresent() {
```

Comment:
Bad practice to implement methods for this way

Better way:
   public boolean isNameInputFieldArePresent() {
        return nameInputField.isElementPresent();
}

Implement all methods with this logic with this way